docs-src/cloud/api-keys/manage-api-keys.md

```diff
@@ -0,0 +1,122 @@
+---
+id: manage-api-keys
+title: Manage API Keys
+sidebar_label: Manage API Keys
+description: Manage API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.
+tags:
+  - how to
+---
+
+Manage API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.
```

Comment:
Manage your personal API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.